stage/dotnet/dotnet-sdk-enterprise-cloud/README.md

```diff
@@ -0,0 +1,111 @@
+# Octokit: .NET SDK
+
+[![Build and test .NET SDK](https://github.com/octokit/dotnet-sdk/actions/workflows/build.yml/badge.svg)](https://github.com/octokit/dotnet-sdk/actions/workflows/build.yml) [![CodeQL](https://github.com/octokit/dotnet-sdk/actions/workflows/github-code-scanning/codeql/badge.svg)](https://github.com/octokit/dotnet-sdk/actions/workflows/github-code-scanning/codeql) [![Publish Release to NuGet](https://github.com/octokit/dotnet-sdk/actions/workflows/publish.yml/badge.svg)](https://github.com/octokit/dotnet-sdk/actions/workflows/publish.yml)
```

Comment:
We should probably add a note about the need to define a host in the headers and with the SDK instantiation itself.

Reply:
Actuallly I think we talked about modifying the middleware to do that for the user.  Does that sound right?

Reply:
I think this should be done automatically for the user by default, and then they have the option of changing it if they'd like. So perhaps both options are appropriate, and that might be best done in a PR to the dotnet-sdk-enterprise-cloud repository after the initial PR is merged. 

There will be some additional cleanup required there, I'm sure.

Reply:
Right now the project builds and I've made a best-effort pass at correcting dotnet-sdk references, but those build badges in the above links for example still reference dotnet-sdk, which need to be changed.